tests/unit/test_agw_preinstall_checks.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import unittest
+from unittest.mock import MagicMock, PropertyMock, call, patch
+
+from magma_access_gateway_installer.agw_installation_errors import (
+    InvalidNumberOfInterfacesError,
+    UnsupportedOSError,
+)
+from magma_access_gateway_installer.agw_preinstall_checks import (
+    AGWInstallerPreinstallChecks,
+)
+
+
+class TestAGWInstallerPreinstallChecks(unittest.TestCase):
+
+    MAGMA_USER_DETAILS = ("magma", "x", 1001, 1001, ",,,", "/home/magma", "/bin/bash")
+    TEST_NETWORK_INTERFACES = ["eth0", "eth1"]
```

Comment:
Why is `MAGMA_USER_DETAILS` a list and `TEST_NETWORK_INTERFACES` a tuple? I don't see the second one being modified anywhere. Is there any specific reason behind that?

Reply:
It's actually the other way around ;)
() - tuple
[] - list
Anyway, something strange happened here, because `MAGMA_USER_DETAILS` should be of `pwd.passwd_struct` type... And it's also not needed here. 
Thanks for pointing that out - I've removed `MAGMA_USER_DETAILS` from here and updated the type in the class it's actually used.